src/kOS/Control/SteeringManager.cs

```diff
@@ -529,7 +529,9 @@ public void UpdateStateVectors()
                 angularAcceleration = new Vector3d(angularAcceleration.x, angularAcceleration.z, angularAcceleration.y);
             }
             
-            momentOfInertia = shared.Vessel.MOI;
+            // TODO: This is a temporary fix for the Moment of Inertial  This can be removed after the next KSP release.
+            //momentOfInertia = shared.Vessel.MOI;
```

Comment:
It might be a good idea to re-phrase this comment now that we don't know if the "next" release will have a better MoI or not.